presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/BingTileFunctions.java

```diff
@@ -371,16 +393,111 @@ private static Point tileXYToLatitudeLongitude(int tileX, int tileY, int zoomLev
         return new Point(longitude, latitude);
     }
 
+    /**
+     * Returns a Bing tile at a given zoom level containing a point at a given latitude and longitude.
+     * Latitude must be within [-85.05112878, 85.05112878] range. Longitude must be within [-180, 180] range.
+     * Zoom levels from 1 to 23 are supported.
+     *
+     * @param latitude
```

Comment:
I don't think these `@param` and `@return` add much value to the javadoc as they are all empty. We can just remove them (same for the javadocs below).